install-git-secrets-hook.sh

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env bash
+
+VERSION="1.2.1"
+BIN_DIRECTORY=$(dirname $(which git))
+BIN_PATH="$BIN_DIRECTORY/git-secrets"
+DOWNLOAD_URL="https://raw.githubusercontent.com/awslabs/git-secrets/$VERSION/git-secrets"
+ME=$(basename "$0")
+
+# TODO: test
+
+if [ ! -f $BIN_PATH ]; then
+    echo "Installing git-secrets v.$VERSION to $BIN_PATH from $DOWNLOAD_URL"
+    wget $DOWNLOAD_URL -O $BIN_PATH
+    RETCODE=$?
+    if [ "$RETCODE" -ne "0" ]; then
+        echo "Please run this script in privileged mode or install git-secrets v. $VERSION manually"
+        echo "wget $DOWNLOAD_URL -O $BIN_PATH"
+        echo "chmod a+x $BIN_PATH"
+        exit 1
+    fi
+    chmod a+x $BIN_PATH
+else
+    echo "git-secrets script already exists: $BIN_PATH"
```

Comment:
worth adding pre message like "installing/checking git-secrets" if we show status here

Reply:
done